bitnami/thanos/values.yaml

```diff
@@ -1834,6 +1847,18 @@ storegateway:
     ## @param storegateway.pdb.maxUnavailable Maximum number/percentage of pods that may be made unavailable
     ##
     maxUnavailable: ""
+  ## Sharded parameters
+  ## @param storegateway.sharded.enabled Enable sharding for Thanos Store Gateway
+  ## @param storegateway.sharded.hashPartitioning.shards Setting hashPartitioning will create multiple store statefulsets based on the number of shards specified using the hashmod of the blocks
+  ## @param storegateway.sharded.timePartitioning [array] Setting time timePartitioning will create multiple store deployments based on the number of partitions
+  ##
+  sharded:
+    enabled: false
+    hashPartitioning:
+      shards: ""
+    timePartitioning:
+      - min: ""
+        max: ""
```

Comment:
I'm missing the `storegateway.sharded.service` parameters, right?

Reply:
yes i will add these , my last push was just to show the direction i was heading

Reply:
pushed